packages/lit-dev-content/site/docs/v3/data/task.md

```diff
@@ -0,0 +1,440 @@
+    },
+  });
+```
+
+{% endswitchable-sample %}
+
+Forwarding the signal to `fetch()` will cause the browser to cancel the network request if the signal is aborted.
+
+You can also check if a signal has been aborted in your task function. It's often useful to race a Promise against a signal:
+
+{% switchable-sample %}
+
+```ts
+  private _task = new Task(this, {
+    task: async ([arg1], {signal}) => {
+      const firstResult = await doSomeWork(arg1);
+      signal.throwIfAborted();
+      const secondResult = await doMoreWork(firstResult);
+      signal.throwIfAborted();
+      return secondResult;
+    },
+  });
+```
+
+```js
+  _task = new Task(this, {
+    task: async ([arg1], {signal}) => {
+      const firstResult = await doSomeWork(arg1);
+      signal.throwIfAborted();
+      const secondResult = await doMoreWork(firstResult);
+      signal.throwIfAborted();
+      return secondResult;
+    },
+  });
+```
+
+{% endswitchable-sample %}
+### Task chaining
+
+Sometimes you want to run one task when another task completes. To do this you can use the value of a task as an argument to the other:
+
+{% switchable-sample %}
+
+```ts
+class MyElement extends LitElement {
+  private _getDataTask = new Task(this, {
+    task: ([dataId]) => getData(dataId),
+    args: () => [this.dataId],
+  });
+
+  private _processDataTask = new Task(this, {
+    task: ([data, param]) => processData(data, param),
+    args: () => [this._getDataTask.value, this.param],
+  });
+}
+```
+
+```js
+class MyElement extends LitElement {
+  _getDataTask = new Task(this, {
+    task: ([dataId]) => getData(dataId),
+    args: () => [this.dataId],
+  });
+
+  _processDataTask = new Task(this, {
+    task: ([data, param]) => processData(data, param),
+    args: () => [this._getDataTask.value, this.param],
+  });
+}
+```
+
+{% endswitchable-sample %}
+
+This can be useful if the tasks have different arguments so that the chained task may run without the first task running again. In this case it'll use the first task like a cache.
+
+You can also often use one task function and await intermediate results:
+
+{% switchable-sample %}
+
+```ts
+class MyElement extends LitElement {
+  private _getDataTask = new Task(this, {
+    task: ([dataId, param]) => {
+      const data = await getData(dataId);
+      return processData(data, param);
+    },
+    args: () => [this.dataId, this.param],
+  });
+}
+```
+
+```js
+class MyElement extends LitElement {
+  _getDataTask = new Task(this, {
+    task: ([dataId, param]) => {
+      const data = await getData(dataId);
+      return processData(data, param);
+    },
+    args: () => [this.dataId, this.param],
+  });
+}
+```
+
+{% endswitchable-sample %}
+
+### Race conditions
+
+A task function can be called while previous task calls are still pending. In this case the `AbortSignal` passed to previous runs will be aborted and the `Task` controller will ignore the result of the previous task runs and only handle the most recent run.
```

Comment:
Maybe this section can be removed? It seems like a duplicate of the `### Aborting tasks` section earlier in the documentation page.